manifests/rhoai/telemetry/otel-collector.yaml

```
@@ -0,0 +1,187 @@
+    metadata:
+      labels:
+        app.kubernetes.io/name: otel-collector
+        app.kubernetes.io/component: telemetry
+        app.kubernetes.io/part-of: training-operator
+    spec:
+      serviceAccountName: training-operator-otel-collector
+      containers:
+      - name: collector
+        image: registry.redhat.io/openshift-observability/otel-collector:latest
+        args:
+          - --config=/conf/otel-config.yaml
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)